app/models/archive.rb

```diff
@@ -10,12 +10,13 @@ def initialize(year:, month:, day: nil, page: 1)
     @year  = year
     @month = month
     @page  = page
+    @day   = day
   end
 
   def articles
     return @articles if defined?(@articles)
 
-    @articles = Article.published.live.root.all
+    @articles = Article.published.live.root.order(title: :desc)
```

Comment:
This will just affect the archive page.  The calendar method is what breaks it up into individual days.  The `ORDER BY` is `ORDER BY published_at DESC, "articles"."title"` so it'll still keep them in the correct published order.

![image](https://cloud.githubusercontent.com/assets/5169/26022160/fab1c6ac-3751-11e7-93bc-edba6f916b18.png)